project/Build.scala

```diff
@@ -96,6 +96,7 @@ object CoroutinesBuild extends MechaRepoBuild {
       "org.scalatest" % "scalatest_2.11" % "2.2.6" % "test",
       "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.2",
       "org.scala-lang" % "scala-reflect" % "2.11.4",
+      "org.scala-lang" % "scala-compiler" % "2.11.4",
```

Comment:
I added it while experimenting with different solutions. Forgot to take it out.

Removed it in 1387ecb.